providers/keap/objectNames.go

```diff
@@ -0,0 +1,20 @@
+package keap
+
+import (
+	"github.com/amp-labs/connectors/internal/datautils"
+	"github.com/amp-labs/connectors/providers/keap/metadata"
+)
+
+// Supported object names can be found under schemas.json.
+var supportedObjectsByRead = metadata.Schemas.ObjectNames() //nolint:gochecknoglobals
+
+// ObjectNameToResponseField maps ObjectName to the response field name which contains that object.
+var ObjectNameToResponseField = datautils.NewDefaultMap(map[string]string{ //nolint:gochecknoglobals
+	"merchants":       "merchant_accounts",
```

Comment:
Would this be handled by the `locatedAt` PR?

Reply:
I moved it to Metadata PR. The answer is yes and no ahaha. Yes, since now on it will be served from the file which conveniently summarizes everything about that object. But this map will live inside the OpenAPI script which produces the file.

Schema.json in this way is a documentation of what is happenning by deep connector implementation. We still need to tell the tool the "exceptions", but it automatically figures out responses with array on top JSON level. If it is an object we need to tell where do we expect the array to be located at. (hence the old name `locatedAt`).